mintpy/utils/readfile.py

```diff
@@ -709,7 +709,9 @@ def read_attribute(fname, datasetName=None, metafile_ext=None):
 
         # FILE_TYPE - k
         py2_mintpy_stack_files = ['interferograms', 'coherence', 'wrapped'] #obsolete mintpy format
-        if any(i in d1_list for i in ['unwrapPhase', 'azimuthOffset']):
+        if 'FILE_TYPE' in atr:
+            k = atr['FILE_TYPE']
```

Comment:
Could we move this here before using the random group/dataset names? That won't change the behavior too much, as I want the function to be less sensitive to the potential wrong FILE_TYPE attribute, which I found myself easily cause when creating an h5 file without specifying the FILE_TYPE.

https://github.com/insarlab/MintPy/blob/9e828af2cecac0f5f05b347a596b4b2736ca1046/mintpy/utils/readfile.py#L728

Reply:
Yes, for sure. I'll change it